lib/components/SegmentedBar.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+    <div
+        class="segmentedbar" 
+        ref="segmentedbar">
+        <!-- List items as tab items if they were given -->
+        <div 
+            ref="listItems"
+            v-if="items && items.length > 0">
+            <div class="tabHeader" v-for="item in items" :key="item">
+                <button
+                    :style="item === chosenTab ? 'border-bottom-width: 5px; border-bottom-color: crimson;':  'border: none;'"
+                    @click="chooseTab(item)"
+                >
+                    <h5>
+                        {{item}}
+                    </h5>
+                </button>
+            </div>
+            <div class="tabContent">
```

Comment:
this should be template/slot driven. Please remove static text content. And transition between tab contents using active/selected classes